Frontend/Gmfctn/src/app/modules/layout/layout.component.html

```diff
@@ -0,0 +1,5 @@
+<app-sidebar [user]="user"></app-sidebar>
+<!--<ng-template #signIn>
```

Comment:
Commented code shouldn't be in the repo until it explains the solution or documenting a piece of code

Reply:
Fixed😎